home_work/hw_11/library/book.py

```diff
@@ -0,0 +1,62 @@
+from loguru import logger as log
+
+
+class Book:
+    def __init__(self, book_name, author, num_pages, isbn):
```

Comment:
`self.reader_name = reader_name` and  `self.reserved_by = reader_name` stores the same information. Seems like you can use only 1 attribute instead of 2.

Reply:
Due to the different logic for reservations and reading, I've decided to keep both attributes.

I've updated the code to avoid confusion and ensure proper tracking of the book's state. Now, the logic for reserving, reading, and returning books should work correctly without any overlap.

Thanks again for your attention to detail and helping improve the code!